bpf/lib/policy.h

```diff
@@ -186,6 +186,17 @@ __policy_can_access(const void *map, struct __ctx_buff *ctx, __u32 local_id,
 				return DROP_POLICY_DENY;
 			return policy->proxy_port;
 		}
+
+		/* Check L4 any port policy */
+		key.dport = 0;
+		policy = map_lookup_elem(map, &key);
+		if (likely(policy)) {
+			account(ctx, policy);
+			*match_type = POLICY_MATCH_L4_ONLY;
+			if (unlikely(policy->deny))
+				return DROP_POLICY_DENY;
+			return CTX_ACT_OK;
+		}
```

Comment:
This code will affect both ingress and egress, so the commit title is incorrect. The other, egress-specific code is for the sockops feature.

I think it's fine to squash both commits.

Reply:
Thank you for your suggestion.